bin/load.js

```diff
@@ -86,6 +86,11 @@ fileParser.add_argument('input', {
     help: 'path to the file/dir to be loaded',
     type: fileExists,
 });
+fileParser.add_argument('--ignoreCache', {
+    action: 'store_true',
+    default: false,
+    help: 'Load the full content, to not check for previously loaded records already in the GraphKB instance',
```

Comment:
does this option overwrite previously loaded records or duplicate them?

Reply:
It will try to create them all and if they already exist it will just ignore the "exists" error and keep going. In cases where there is a difference to the original record it will do an upsert

Reply:
am I interpreting this wrong, or is this option only being applied to the ncit loader?

Reply:
this option is applied to the "file" subparser so that would affect any data that is loaded via that subparser. Currently the only one that uses the cache is NCIT. In the future there may be others

Reply:
If it helps, the js library I am using here is a port of a python one so you can look at the python docs and it should behave the same for the most part: https://docs.python.org/3/library/argparse.html#argparse.ArgumentParser.add_subparsers

Reply:
re "file" subparser, i see, makes sense.